js/src/main/scala/pl/metastack/metaweb/DiffSupport.scala

```diff
@@ -216,19 +216,11 @@ trait DiffSupport extends DiffSupportLowPrio {
   }
 
   /** TODO Introduce BooleanAttribute and StringAttribute for better type-safety? */
-  implicit class AttributeExtensions[T <: tree.Tag, G, S](attribute: Attribute[T, G, S]) {
+  implicit class AttributeExtensions[T <: tree.Tag, G](attribute: Attribute[T, G, _]) {
```

Comment:
We don't lose any type information here. I removed `update()` which referenced `S`. Now this type parameter became redundant.